src/resources/views/crud/fields/select2_from_ajax_multiple.blade.php

```diff
@@ -94,28 +94,8 @@ function bpFieldInitSelect2FromAjaxMultipleElement(element) {
         var $allowClear = element.attr('data-column-nullable') == 'true' ? true : false;
         var $dependencies = JSON.parse(element.attr('data-dependencies'));
         var $ajaxDelay = element.attr('data-ajax-delay');
-        var $selectedOptions = typeof element.attr('data-selected-options') === 'string' ? JSON.parse(element.attr('data-selected-options')) : JSON.parse("[]");
```

Comment:
Same thing happens here - right? Instead of populating the selected options using JS, we now do it in PHP?

Reply:
Like I said in other reply, this was only coded to support repeatable without BC in 4.1. I think there is no need to keep doing that if we don't need it.

Reply:
Ok, I was skeptical at first but... I really can't think of a reason we can't go back to defining things in PHP now... I think you're right. Closing.